### ISSUE TITLE
Any detail report about CSIG-FAT-AI competition?

### ISSUE THREAD

**Issue by luameows** [Tencent/TFace#26]
Congratulation for ur team winning the second place of CFAT competition. I'm very instrested in your work about making the feature distribution of masked-face and non-masked-face consistent.  In my practice, masked faces' feature has very large gap with non-masked faces. This may cause accuracy decrease when train masked-face and nomasked-face together. I have tried DDL to try to decrease the gap between these two datases but the result is less effective.
Can you provide more information about this task?

**Comment by hitspring2015** [comment]
DDL is too strict for the constraints between masks and non-masks, just like forcing masks and non-masks to use the same margin loss for constraint. I think there should be some softer way to constrain the two to ensure that the accuracy of normal samples does not decrease.

**Comment by hitspring2015** [comment]
DDL is one of our most effective methods, but it may be more sensitive to parameters, you can try to change some hyperparameters

**Comment by luameows** [comment]
I have tried another way by using memory-bank based method, and add few masked-faces. After finetune with curricular-face, the accuracy got better than ddl or without memory-bank.
